test/units/staking/stakeManager/StakeManager.test.js

```diff
@@ -1192,6 +1192,127 @@ contract('StakeManager', async function (accounts) {
     })
   })
 
+  describe.only('checkSignature() with votes', function () {
```

Comment:
Almost there, now move this into existing checkSignatures and rename it to "with votes" and dont forget to remove .only()

Reply:
done